x/axelarnet/keeper/msg_server.go

```diff
@@ -70,15 +70,15 @@ func (s msgServer) CallContract(c context.Context, req *types.CallContractReques
 	txHash := sha256.Sum256(ctx.TxBytes())
 	payloadHash := crypto.Keccak256(req.Payload)
 
-	msg := nexus.NewGeneralMessage(s.nexus.GenerateMessageID(ctx, hex.EncodeToString(txHash[:])), exported.Axelarnet.Name, req.Sender.String(), req.Chain, req.ContractAddress, payloadHash, nexus.Approved, nil)
+	msg := nexus.NewGeneralMessage(s.nexus.GenerateMessageID(ctx, hex.EncodeToString(txHash[:])), exported.Axelarnet.Name, req.Sender.String(), req.Chain, req.ContractAddress, payloadHash, nexus.Sent, nil)
```

Comment:
for context, i preferred using Sent status here directly instead of Approved to simplify the state machine in nexus. But the naming isn't great :/ Logically, for EVM chains, the statuses are more so, pending, approved, or failed. But since the statuses are shared with IBC for cosmos, we end up using Sent, Executed, or Failed.

Open to renaming suggestions for the statuses, but could do that in a separate PR